frontends/web/src/api/bitbox02.ts

```diff
@@ -165,6 +166,7 @@ export const verifyChannelHash = (
 
 export const setPassword = (
   deviceID: string,
+  seed_len: 16 | 32,
```

Comment:
maybe change to camleCase, since other args also use it?

Reply:
right! done